synapse/storage/database.py

```diff
@@ -718,7 +718,9 @@ def inner_func(conn, *args, **kwargs):
             # pool).
             assert not self.engine.in_transaction(conn)
 
-            with LoggingContext("runWithConnection", parent_context) as context:
+            with LoggingContext(
+                str(curr_context), parent_context=parent_context
```

Comment:
`str(curr_context)` will either be `parent_context.name` or `"sentinel"`. We do this in a few other spots so seems reasonable to do here too.

We could add a suffix, but in that case should be using `nested_logging_context`. 🤷

Reply:
I also couldn't see any other `LoggingContext` that need to be changed